scipy/stats/tests/test_stats.py

```diff
@@ -5206,6 +5230,15 @@ def convert(t, p, alt):
     assert_allclose(p, converter(tr, pr, 'greater'), rtol=1e-14)
 
 
+def test_ttest_ind_scalar():
+    # test scalars
+    with suppress_warnings() as sup, np.errstate(invalid="ignore"):
+        sup.filter(RuntimeWarning, "Degrees of freedom <= 0 for slice")
+        t, p = stats.ttest_ind(4., 3.)
+    assert_(np.isnan(t))
+    assert_(np.isnan(p))
```

Comment:
optional but nice to modernise at the same time
```suggestion
    assert np.isnan(t)
    assert np.isnan(p)
```

Reply:
Please go ahead and add this in the `ttest_rel` PR.